core/shared/src/main/scala/com/monovore/decline/opts.scala

```diff
@@ -99,6 +99,8 @@ object Opts {
       override def combineK[A](x: Opts[A], y: Opts[A]): Opts[A] = Opts.OrElse(x, y)
     }
 
+  implicit def monoid[A]: Monoid[Opts[A]] = alternative.algebra[A]
```

Comment:
The shame of it is that this isn't a low priority default if `Alternative[F]` exists...

but anyway, this looks good to me.